### DISCUSSION TITLE
Perform a query with LIKE clause

### DISCUSSION THREAD

**Discussion by own2pwn** [uptrace/bun#558] [Q&A] [answered]
How do you perform a query with LIKE clause?
Escaping `%` with `\\` doesn't work.
The only way I found working: `db.NewSelect().Model(&users).Where("login LIKE '%' || ? || '%'", param)`
It seems ugly. Is there any other way?

**Comment by vmihailenco** [comment, marked as answer]
`Where("login LIKE ?", "%"+param+"%")` should work
